plugins/recruit-resources/src/components/CreateCandidate.svelte

```diff
@@ -153,15 +154,7 @@
   </div>
 
   <div class="flex-row-center channels">
-    {#if !object.channels || object.channels.length === 0}
-      <CircleButton icon={IconAdd} size={'small'} transparent on:click={(ev) => showPopup(contact.component.SocialEditor, { values: object.channels ?? [] }, ev.target, (result) => { object.channels = result })} />
-      <span><Label label={presentation.string.AddSocialLinks} /></span>
-    {:else}
-      <Channels value={object.channels} size={'small'} />
-      <div class="ml-1">
-        <CircleButton icon={contact.icon.Edit} size={'small'} transparent on:click={(ev) => showPopup(contact.component.SocialEditor, { values: object.channels ?? [] }, ev.target, (result) => { object.channels = result })} />
-      </div>
-    {/if}
+    <ChannelsEditor attachedTo={candidateId} attachedClass={contact.class.Person} />
```

Comment:
What it we press cancel, channel will be left behind and not removed.

Reply:
Yes. Like a resume for example ;) ... I think we need call `deleteObject` before close, but I'm not sure about it.

Reply:
I think we should not create anythink before candidate is created. Because transactions will left not removed if you will call deleteObject.